includes/Database_Upgrader.php

```diff
@@ -59,6 +59,7 @@ public function init() {
 		$routines = [
 			'1.0.0' => 'upgrade_1',
 			'2.0.0' => 'v_2_replace_conic_style_presets',
+			'3.0.0' => 'v_3_remove_old_text_styles',
```

Comment:
We are trying to use [Semantic Versioning](https://semver.org/). 3.0.0 would be breaking change. Why not 2.0.1 or 2.1.0?

Reply:
I just followed the previous migration logic because it does something similar.
But maybe 2.0.1 will be better because we primarily want to fix the data.